src/storage/src/hummock/shared_buffer/mod.rs

```diff
@@ -233,7 +234,9 @@ impl SharedBuffer {
                 UncommittedData::Batch(batch) => {
                     range_overlap(key_range, batch.start_user_key(), batch.end_user_key())
                 }
-                UncommittedData::Sst((_, info)) => filter_single_sst(info, key_range),
+                UncommittedData::Sst((_, info)) => {
+                    filter_single_sst(info, TableId::default(), key_range)
```

Comment:
why dont we need to provide the TableID ?

Reply:
> why dont we need to provide the TableID ?

fixed